app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -124,12 +124,23 @@ def analyze
                           msg % [options[:image_full_name], options[:docker_image_id][0..11], actual[0..11]],
                           'error')
     end
+
+    collect_compliance_data(image)
+
     image.scan_metadata(SCAN_CATEGORIES,
                         "taskid" => jobid,
                         "host"   => MiqServer.my_server,
                         "args"   => [YAML.dump(scan_args)])
   end
 
+  def collect_compliance_data(image)
+    # image.openscap_result.try(:destroy)
```

Comment:
@durandom I've added a comment. 
The commented code Is nice for me since I can run tests at my local machine by only un commenting the code[1], but I guess I can remove it at this point @simon3z ?

[1] As Travis does not, and will never have the needed binary for test on that specific code.